services/frontend-v3/package.json

```diff
@@ -75,6 +73,8 @@
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-react": "7.21.4",
     "eslint-plugin-react-hooks": "4.1.2",
-    "prettier": "2.1.2"
+    "prettier": "2.1.2",
+    "webpack-bundle-analyzer": "^3.9.0",
+    "webpackbar": "^4.0.0"
```

Comment:
can we fix the version (remove the "^").

Reply:
done